### PR TITLE
SW-615_fix_final_sw57_issues

### PR DESCRIPTION
**Pull request by TheKeymaster**

### CODE REVIEW

File: FinSearchUnified/Tests/BusinessLogic/Models/FindologicArticleModelTest.php [hunk 1]
```diff
@@ -46,8 +47,17 @@ protected function tearDown()
     private function createTestProduct(array $testProductConfiguration)
     {
         try {
-            /** @var ArticleResource $resource */
-            $resource = Manager::getResource('Article');
+            $manager = Shopware()->Models();
+
+            $resource = new ArticleResource();
+            $resource->setManager($manager);
```
Comment (dblaichinger) [inline FinSearchUnified/Tests/BusinessLogic/Models/FindologicArticleModelTest.php:53]: Shouldn't this happen after `$manager->create()`?

File: FinSearchUnified/Tests/Functional/Controllers/Frontend/FinSearchUnified_Tests_Controllers_Frontend_SearchTest.php [hunk 2]
```diff
@@ -160,6 +164,8 @@ public function testSmartDidYouMeanSuggestionsAreDisplayed(
         array $expectedText,
         $expectedLink
     ) {
+        $this->markTestSkipped('Skipped due to issues with Shopware 5.7');
```
Comment (dblaichinger) [inline FinSearchUnified/Tests/Functional/Controllers/Frontend/FinSearchUnified_Tests_Controllers_Frontend_SearchTest.php:167]:
```suggestion
        $this->markTestSkipped('Skipped due to issues with Shopware 5.7 - see SW-622');
```

File: FinSearchUnified/Tests/Subscriber/WidgetsTest.php [hunk 3]
```diff
@@ -289,6 +306,8 @@ public function homePageProvider()
      */
     public function testHomePage($referer)
     {
+        $this->markTestSkipped('Skipped due to issues with Shopware 5.7');
```
Comment (dblaichinger) [inline FinSearchUnified/Tests/Subscriber/WidgetsTest.php:309]:
```suggestion
        $this->markTestSkipped('Skipped due to issues with Shopware 5.7 - see SW-622');
```

**Review by dblaichinger — CHANGES_REQUESTED** [review]

File: FinSearchUnified/Tests/BusinessLogic/Models/FindologicArticleModelTest.php (hunk 1 above)
Comment (TheKeymaster) [reply FinSearchUnified/Tests/BusinessLogic/Models/FindologicArticleModelTest.php:53]: Doesn't really matter as instances are passed by reference and not by value.

File: FinSearchUnified/Resources/views/frontend/listing/filter/_includes/filter-color-selection.tpl [hunk 4]
```diff
@@ -27,5 +27,7 @@
         {else}
             <img class="filter-panel--media-image color--filter-item" title="{$option->getId()|escape:'htmlall'}" src="/custom/plugins/FinSearchUnified/Resources/views/frontend/_public/src/img/no-picture.png" alt="{$option->getId()|escape:'htmlall'}">
         {/if}
+
+        <span class="color--filter-name">{$option->getId()|escape:'htmlall'}</span>
```
[the remark this answers is not in the document]
Comment (TheKeymaster) [reply FinSearchUnified/Resources/views/frontend/listing/filter/_includes/filter-color-selection.tpl:31]: Attached an image to my comment on YouTrack.

**Pull request merged**